README.md

```diff
@@ -1,6 +1,7 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Portfolio
+Here is the web page link: https://fherrerao.github.io/Portfolio/
```

Comment:
## [OPTIONAL]

- [ ] Kindly your README  would look more professional if you include the section of the ***live demo*** as the microverse README template is structured and removing it from the description section. Just like this :point_down: 
![Screenshot from 2021-12-06 12-43-06](https://user-images.githubusercontent.com/65068771/144832871-62402c3f-108d-4d10-aade-d34d4034b152.png)
- [ ] would recommend also to use GitHub editor feature by putting your demo link inside a hyperlink. Just like [link’s name](https//yourlinkhere)

Reply:
Thanks, I just done your suggestion.